tutorials/hana-clients-choose-hana-instance/hana-clients-choose-hana-instance.md

```diff
@@ -77,31 +77,33 @@ Here are a few benefits of using SAP HANA Cloud:
 
 1. >To complete the tutorials in this mission, an SAP HANA instance is needed. Step 3 and 5 in this tutorial provide two different, free options that can be used to set up an SAP HANA instance.  Only one of these steps needs to be completed if you currently do not have access to an SAP HANA instance.
 
-Continue on with this tutorial once you have created an SAP HANA Cloud trial instance as shown below.
+    Continue on with this tutorial once you have created an SAP HANA Cloud trial instance as shown below.
 
-!![SAP HANA Cloud Trial instance](hana-cloud-instance.png)
+    !![SAP HANA Cloud Trial instance](hana-cloud-instance.png)
 
 
-The instructions on how to setup a free SAP HANA Cloud trial within the SAP Cloud Platform are well covered in a number of other sources listed below.  
+    The instructions on how to setup a free SAP HANA Cloud trial within the SAP Cloud Platform are well covered in a number of other sources listed below.  
 
-  * [Help Thomas Get Started with SAP HANA](https://developers.sap.com/tutorials/hana-trial-advanced-analytics.html)
+    * [Help Thomas Get Started with SAP HANA](https://developers.sap.com/tutorials/hana-trial-advanced-analytics.html)
```

Comment:
Please change the format of links to tutorials as described in the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqgno1se).